criu/pie/restorer.c

```diff
@@ -1327,17 +1327,24 @@ static int fd_poll(int inotify_fd)
  */
 #define EVENT_BUFF_SIZE ((sizeof(struct inotify_event) + PATH_MAX))
 
+/* We can get interrupted in poll, so try some more */
+#define MAX_POLL_RETRY 5
```

Comment:
Nack. We don't expect any external signals, so we have to know when ppoll can be interrupted and how many times.